.gitignore

```
@@ -42,3 +42,11 @@ package-lock.json
 
 yarn.lock
 
+# ignore service workers
+# Auto Generated PWA files
+**/public/sw.js
+**/public/workbox-*.js
+**/public/worker-*.js
+**/public/sw.js.map
+**/public/workbox-*.js.map
+**/public/worker-*.js.map
```

I forgot to tell that add these files & also, delete 

```
sw.js
sw.js.map
workbox-3576cac3.js
workbox-3576cac3.js.map
```

```suggestion
sw.js
sw.js.map
workbox-3576cac3.js
workbox-3576cac3.js.map
```

---

@SubhamB2003 please check this